markdowns/10_Knowledge_Graph.md

```diff
@@ -3,7 +3,7 @@ layout: tutorial
 colab: https://colab.research.google.com/github/deepset-ai/haystack-tutorials/blob/main/tutorials/10_Knowledge_Graph.ipynb
 toc: True
 title: "Question Answering on a Knowledge Graph"
-last_updated: 2022-10-12
+last_updated: 2022-11-23
```

Comment:
do you know why this was updated?

Reply:
Yes because I tested this one without adding the 'slug' so if you don't have the slug it generates a .md file with the name of the .ipynb